backend/backend/settings/base.py

```diff
@@ -95,7 +95,7 @@ def get_required_setting(
 WEB_APP_ORIGIN_URL = os.environ.get("WEB_APP_ORIGIN_URL", "http://localhost:3000")
 
 LOGIN_NEXT_URL = os.environ.get("LOGIN_NEXT_URL", "http://localhost:3000/org")
-LANDING_URL = os.environ.get("LANDING_URL", "http://localhost:3000/landing")
+LANDING_URL = os.environ.get("LANDING_U`RL", "http://localhost:3000/landing")
```

Comment:
Is that backtick in `LANDING_U`RL`  indentioanally added?

Reply:
My bad, Thanks for pointing out